sonaranalyzer-dotnet/build/rspec.ps1

```diff
@@ -242,5 +243,9 @@ if ($className -And $ruleKey) {
        UpdateTestEntry $csRuleData
     }
 
-    Remove-Item -Recurse -Force "${PSScriptRoot}\\..\\.vs"
```

Comment:
That was my initial idea, but that would also ignore problems when files are in use, which is useful to err on. I didn't find an easy way to only ignore "not found" problem.